pkg/hubble/peer/serviceoption/option.go

```diff
@@ -5,13 +5,28 @@ package serviceoption
 
 // Options stores all the configuration values for the peer service.
 type Options struct {
-	MaxSendBufferSize int
-	WithoutTLSInfo    bool
+	MaxSendBufferSize       int
+	WithoutTLSInfo          bool
+	AddressFamilyPreference AddressFamilyPreference
 }
 
 // Option customizes the peer service's configuration.
 type Option func(o *Options)
 
+type addressFamily uint
+
+const (
+	AddressFamilyIPv4 addressFamily = 4
+	AddressFamilyIPv6 addressFamily = 6
+)
+
+type AddressFamilyPreference []addressFamily
```

Comment:
nitpicking: it's a bit odd to expose `type`/`const` while the underlying type `addressFamily` is private.

```suggestion
type AddressFamily uint

const (
	AddressFamilyIPv4 AddressFamily = 4
	AddressFamilyIPv6 AddressFamily = 6
)

type AddressFamilyPreference []AddressFamily
```

Reply:
Thanks for the remark! I've adjusted the code accordingly and pushed a new version.